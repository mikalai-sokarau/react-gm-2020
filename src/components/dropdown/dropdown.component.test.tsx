import React from 'react';
import renderer from 'react-test-renderer';

import Dropdown from '@app/components/Dropdown/Dropdown.component';
import sortOptions from '@app/components/preferenceBar/preferenceBar.model';

describe('Logo', () => {
  it('Correctly renders Logo component', () => {
    const dropdown = renderer
      .create(
        <Dropdown
          title="dropdown test"
          options={sortOptions}
          onSortOptionClick={() => 'clicked'}
        />,
      )
      .toJSON();

    expect(dropdown).toMatchSnapshot();
  });
});
