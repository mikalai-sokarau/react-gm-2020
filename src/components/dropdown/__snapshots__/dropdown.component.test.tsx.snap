// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Correctly renders Logo component 1`] = `
<div
  onMouseLeave={[Function]}
>
  <span
    className="sortTitle-0-2-4"
  >
    sort by
  </span>
  <div
    className="dropdownContainer-0-2-2"
    onClick={[Function]}
  >
    <div
      className="dropdownTitle-0-2-3"
      role="menu"
      tabIndex={0}
    >
      dropdown test
    </div>
    <ul
      className="optionsList-0-2-5"
    >
      <li
        className="option-0-2-1"
        onClick={[Function]}
        role="menuitem"
        tabIndex={0}
      >
        title
      </li>
      <li
        className="option-0-2-1"
        onClick={[Function]}
        role="menuitem"
        tabIndex={0}
      >
        release date
      </li>
      <li
        className="option-0-2-1"
        onClick={[Function]}
        role="menuitem"
        tabIndex={0}
      >
        rating
      </li>
    </ul>
  </div>
</div>
`;
